### PR TITLE
TinyTask: Add version 1.72

### PR DESCRIPTION
**Pull request by wenmin92**
[TinyTask](https://www.tinytask.net/) is a minimalist Windows automation app you can use to record and repeat actions.

### CODE REVIEW

File: bucket/tinytask.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://www.tinytask.net/",
+    "description": "A minimalist Windows automation app you can use to record and repeat actions.",
```
Comment (Ash258) [inline bucket/tinytask.json]:
```suggestion
    "description": "A minimalist Windows automation app used for recording and repeating actions.",
```

File: bucket/tinytask.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.tinytask.net/faq.php"
+    },
+    "url": "https://www.tinytask.net/TinyTask.exe",
+    "version": "1.72",
```
Comment (Ash258) [inline bucket/tinytask.json]: Move version above license

```suggestion
```

File: bucket/tinytask.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://www.tinytask.net/",
+    "description": "A minimalist Windows automation app you can use to record and repeat actions.",
+    "license": {
```
Comment (Ash258) [inline bucket/tinytask.json]:
```suggestion
    "version": "1.72",
    "license": {
```

File: bucket/tinytask.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "hash": "9957f2b0e80a03421ff11ab680cb07d111409a03c41540db5a6998ff685f6265",
+    "shortcuts": [
+        [
+            "TinyTask.exe",
+            "TinyTask"
+        ]
+    ],
+    "pre_install": "if (!(Test-Path \"$persist_dir\\TinyTask.ini\")) { New-Item \"$dir\\TinyTask.ini\" -ItemType File | Out-Null }",
+    "persist": "TinyTask.ini",
+    "checkver": "version ([\\w.]+)",
```
Comment (Ash258) [inline bucket/tinytask.json]: `\w` is pretty overkil for just numbers

```suggestion
    "checkver": "version ([\\d.]+)",
```

**Comment by r15ch13** [comment]
What a nice program! 👍 

Small changes:
- Added `bin` property
- Use URL with version number from https://www.tinytask.net/revision_history.php
```json
{
    "homepage": "https://www.tinytask.net/",
    "description": "A minimalist Windows automation app used for recording and repeating actions.",
    "version": "1.72",
    "license": {
        "identifier": "Freeware",
        "url": "https://www.tinytask.net/faq.php"
    },
    "url": "https://www.tinytask.net/tinytask_172.exe#/TinyTask.exe",
    "hash": "9957f2b0e80a03421ff11ab680cb07d111409a03c41540db5a6998ff685f6265",
    "bin": "TinyTask.exe",
    "shortcuts": [
        [
            "TinyTask.exe",
            "TinyTask"
        ]
    ],
    "pre_install": "if (!(Test-Path \"$persist_dir\\TinyTask.ini\")) { New-Item \"$dir\\TinyTask.ini\" -ItemType File | Out-Null }",
    "persist": "TinyTask.ini",
    "checkver": "version ([\\d.]+)",
    "autoupdate": {
        "url": "https://www.tinytask.net/tinytask_$cleanVersion.exe#/TinyTask.exe"
    }
}
```

**Pull request merged**